pkg/controllers/mpi_job_controller.go

```diff
@@ -940,18 +934,26 @@ func newLauncherRole(mpiJob *kubeflow.MPIJob, workerReplicas int) *rbacv1.Role {
 	}
 }
 
+func getLaunchName(mpiJob *kubeflow.MPIJob) string {
```

Comment:
Consider moving these functions to the bottom since the functions are being used both above and below their definitions.